src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -12,7 +12,10 @@ import {ANONYMOUS} from '../constant';
 import {AnonUserInfo} from '../../../../types/service/AnonProfile.type';
 import {ChannelData, ChannelType} from '../../../../types/repo/ChannelData';
 import {DELETED_MESSAGE_TEXT, MESSAGE_TYPE_DELETED} from '../../../utils/constants';
+import {DatabaseOperationLabel} from '../../../core/queue/DatabaseQueue';
+import {QueueJobPriority} from '../../../core/queue/BaseQueue';
 import {getChannelListInfo, getChannelMembers} from '../../../utils/string/StringUtils';
+import {getLatestTopicPost} from '../../../service/topics';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `DatabaseOperationLabel` and `QueueJobPriority` is new, as well as the function `getLatestTopicPost`. Ensure that these are correctly implemented and used in the code.
```diff
+ import {DatabaseOperationLabel} from '../../../core/queue/DatabaseQueue';
+ import {QueueJobPriority} from '../../../core/queue/BaseQueue';
+ import {getLatestTopicPost} from '../../../service/topics';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->